template/cordova/lib/MSBuildTools.js

```diff
@@ -93,6 +93,20 @@ module.exports.findAllAvailableVersions = function () {
             return !!item;
         });
     });
+}
+
+module.exports.findAllAvailableVersions = function () {
+    // CB-11548 use VSINSTALLDIR environment if defined to find MSBuild. If VSINSTALLDIR
+    // is not specified or incorrect - fall back to default discovery mechanism.
```

Comment:
It is quite likely that `VSINSTALLDIR` is specified and *correct*, but just doesn't contain the msbuild path we're looking for (this will be the case for previous versions of VS, for example). So I think it would be clearer to say here something like "not specified or doesn't contain the MSBuild path we are looking for".